src/client/components/StaffViewRoute/index.js

```diff
@@ -0,0 +1,115 @@
+// This file is identical to RetroCertsRoute other than the section
+// near `if (isProdEnvironment)` which restricts access to Staff View
+import PropTypes from "prop-types";
+import { Route, Redirect } from "react-router-dom";
+import React from "react";
+import { userDataPropType, setUserDataPropType } from "../../commonPropTypes";
+import AUTH_STRINGS from "../../../data/authStrings";
+import routes from "../../../data/routes";
+import SessionTimer, { clearAuthToken } from "../SessionTimer";
+import ScrollToTop from "../ScrollToTop";
+
+function userIsAuthenticated() {
+  return !!sessionStorage.getItem(AUTH_STRINGS.authToken);
+}
+
+function userDataIsSet(userData) {
+  return !!userData.weeksToCertify;
+}
+
+/**
+ * A component that wraps authorized page components.
+ *
+ * This does the following:
+ * - Passes props from the Route down to the page component.
+ * - Adds the session timer to each page
+ *
+ * @returns {React.ReactElement}
+ */
+function AuthorizedPageWrapper(props) {
+  const { pageComponent: Component, pageProps, computedMatch } = props;
+
+  return (
+    <React.Fragment>
+      <ScrollToTop />
+      <Component routeComputedMatch={computedMatch} {...pageProps} />
+      <SessionTimer
+        action="startOrUpdate"
+        setUserData={pageProps.setUserData}
+      />
+    </React.Fragment>
+  );
+}
+
+AuthorizedPageWrapper.propTypes = {
+  pageComponent: PropTypes.func.isRequired,
+  pageProps: PropTypes.shape({
+    userData: userDataPropType,
+    setUserData: setUserDataPropType,
+  }),
+  computedMatch: PropTypes.object.isRequired,
+};
+
+function StaffViewRoute(props) {
+  const {
+    pageComponent: Component,
+    pageProps,
+    requiresAuthentication,
+    ...routeProps
+  } = props;
+
+  let routeChild = <div>Loading...</div>;
+  const hostname = window.location.hostname;
+  const isProdEnvironment = hostname === "unemployment.edd.ca.gov";
+  if (isProdEnvironment) {
+    // Staff view should currently not load on production at all,
+    routeChild = <Redirect to={routes.retroCertsAuth} push />;
+  } else {
+    if (!requiresAuthentication) {
+      routeChild = <Component {...pageProps} />;
+    } else if (userIsAuthenticated() && userDataIsSet(pageProps.userData)) {
```

Comment:
This is probably just `} else if (userDataIsSet(pageProps.userData)) {`, since there's no authToken.